templates/etc/sysconfig/docker-storage.erb

```diff
@@ -23,5 +26,9 @@ DOCKER_STORAGE_OPTIONS="<% -%>
 <% if @dm_loopmetadatasize %> --storage-opt dm.loopmetadatasize=<%= @dm_loopmetadatasize %><% end -%>
 <% if @dm_datadev %> --storage-opt dm.datadev=<%= @dm_datadev %><% end -%>
 <% if @dm_metadatadev %> --storage-opt dm.metadatadev=<%= @dm_metadatadev %><% end -%>
```

Comment:
@DjxDeaf are you talking about the `dm_datadev` and `dm_metadatadev` options in the template file, because those variables already existed prior to this PR.  If not, I think I missed the problem you were stating about the template files.